app/api/events.py

```diff
@@ -605,6 +585,12 @@ def start_export_tasks(event):
     create_export_job(task_pentabarf.id, event_id)
 
 
+def start_image_resizing_tasks(event, data):
```

Comment:
Just getting the `data['original_image_url']` instead of `data` as a whole will be much better. There's no use of data and also celery needs to serialize and deserialize complex objects using JSON or pickle making it an expensive call to pass big objects rather than just strings